app-misc/figlet/figlet-222.recipe

```diff
@@ -14,12 +14,13 @@ CHECKSUM_SHA256="21f8f9e9b2190fbc0f801fb4704355e5d73d36c55670272cd08cad3408d1f5f
 SOURCE_DIR="figlet222"
 
 ARCHITECTURES="x86_gcc2 x86 x86_64"
+
 PROVIDES="
     figlet = $portVersion
-    cmd:figlet =$portVersion
     cmd:chkfont
-    cmd:showfigfonts
+    cmd:figlet =$portVersion
```

Comment:
I'd leave the cmd:figlet =$portVersion up in the 2nd line to have the "main" command on top.